### PR TITLE
Set assignees state to list assignments

### PR DESCRIPTION
**Pull request by tmikeschu**
@Tman22 I'm sure there is a better Redux way to do this, but this makes it so I can actually see my assignments.

### CODE REVIEW

File: src/Components/Main/AssigneeList/AssigneeList.js [hunk 1]
```diff
@@ -3,25 +3,27 @@ import Assignee from './Assignee/Assignee';
...
-  //     .then(data => this.setState({ assignees: data }))
-  // }
+  constructor(props) {
+    super(props);
+    this.state = { assignees: [] }
+  }
+  
+  componentDidMount() {
+    console.log('why', this.props);
+    this.props.user.id && fetch(`https://johariwindowapi.herokuapp.com/api/v1/users/${this.props.user.id}/assignments`)
```
Comment (Tman22) [inline src/Components/Main/AssigneeList/AssigneeList.js]: You're right! So I brought in thunk, which allows us to use fetch calls inside our actions.

**Comment by Tman22** [comment]
Would you want me to whip that up or do you want to take a crack at it?

**Comment by tmikeschu** [comment]
Wouldn't mind the reps! I'll timebox it and leave it to you if I hit a wall?

**Comment by Tman22** [comment]
Sounds good! Let me know if you want some help

**Comment by Tman22** [comment]
@tmikeschu I forgot to mention there are reducers and actions already made for this. You might want to add it on the fetchUser action. After the resolve of the current Axios/Promise which should return the user.id

**Comment by tmikeschu** [comment]
Cool! I'm still pretty lost after 25 minutes, so I'm either down to pair tomorrow or you can just do it and I'll read your PR carefully 😉 @Tman22

**Comment by Tman22** [comment]
Haha I might just see what I get done, I'll let you know here in a little bit!

**Comment by Tman22** [comment]
Feel free to pull this down and check it out. Ready to merge whenever.

File: src/Components/Main/AssigneeList/AssigneeList.js [hunk 2]
```diff
@@ -3,25 +3,14 @@ import Assignee from './Assignee/Assignee';
...
-  // componentDidMount() {
-  //   console.log('why', this.props);
-  //   fetch(`https://johariwindowapi.herokuapp.com/api/v1/users/${this.props.user.id}/assignments`)
-  //     .then(result => result.json())
-  //     .then(data => this.setState({ assignees: data }))
-  // }
 
   eachAssignee(assignee, i) {
-    this.props.assignees.map((assignee, i) => {
+    return this.props.assignees.map((assignee, i) => {
```
Comment (tmikeschu) [inline src/Components/Main/AssigneeList/AssigneeList.js:8]: Where are these props getting assigned originally?

Comment (tmikeschu) [reply src/Components/Main/AssigneeList/AssigneeList.js:8]: Thanks for chatting about this ^ today 👍

**Pull request merged**